<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Please Enter Text ...</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="txtFromTime">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="txtToTime">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSet">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFileName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblData">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>256</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Text</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>From</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Until</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSelectFile">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>270</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBold">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkItalic">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkUnderline">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
